### PR TITLE
Nm33merge

### PR DESCRIPTION
**Pull request by pierromond**

### CODE REVIEW

File: wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/NoiseModelling/Road_Emission_from_Traffic.groovy [hunk 1]
```diff
@@ -176,21 +186,21 @@ def exec(Connection connection, input) {
 
     // Get Class to compute LW
     LDENConfig ldenConfig = new LDENConfig(LDENConfig.INPUT_MODE.INPUT_MODE_TRAFFIC_FLOW)
+    ldenConfig.setCoefficientVersion(2)
```
Comment (nicolas-f) [inline wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/NoiseModelling/Road_Emission_from_Traffic.groovy:189]: oh ?

**Comment by nicolas-f** [comment]
Ok for me. 
ldenConfig.setCoefficientVersion(2) was added. Is it expected ?

File: wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/NoiseModelling/Road_Emission_from_Traffic.groovy (hunk 1 above)
Comment (pierromond) [reply wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/NoiseModelling/Road_Emission_from_Traffic.groovy:189]: Yes ! it is ok.

**Comment by pierromond** [comment]
Yes, expected. By default, I prefer that the user use the coefficient of Cnossos 2015

**Pull request merged**